src/TLuaInterpreter.cpp

```diff
@@ -12041,7 +12041,7 @@ int TLuaInterpreter::setServerEncoding(lua_State * L)
     }
     else {
         lua_pushnil(L);
-        lua_pushfstring(L, "setServerEncoding: bad argument #1 value %s",
+        lua_pushfstring(L, "Error: %s.",
```

Comment:
Not even an `Error` - it's obvious it is an error, it is the 2nd return which can only be an error! _just_ the actual error message itself please, leave the styling to the API.